ctsit/mobile_api_request_to_survey
# Develop

Note: This PR is related to issue [RCGP-114](https://jira.ctsi.ufl.edu/browse/RCGP-114)

## Testing guide
- After configuring the module with the required a URL, test that the "Request API Token" button on MobileApp/index.php redirects to `https://redcap.ctsi.ufl.edu/redcap/surveys/?s=9NPEPAPHJE`.
- Add an optional text (with html formatting) in the module configuration and check that the text appears on the MobileApp/index.php page.
